webapp/components/_new/generic/UserAvatar/UserAvatar.vue

```diff
@@ -3,9 +3,11 @@
     <span class="initials">{{ userInitials }}</span>
     <base-icon v-if="isAnonymous" name="eye-slash" />
     <img
-      v-else
+      v-if="user && user.avatar"
       :src="user.avatar | proxyApiUrl"
       class="image"
+      :alt="user.name"
+      :title="user.name"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T12:30:58Z" title="Tuesday, April 7th 2020, 2:30:58 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -3,9 +3,11 @@
     <span class="initials">{{ userInitials }}</span>
     <base-icon v-if="isAnonymous" name="eye-slash" />
     <img
-      v-else
+      v-if="user && user.avatar"
       :src="user.avatar | proxyApiUrl"
       class="image"
+      :alt="user.name"
+      :title="user.name"
```


I like this a lot! :+1: 🤓

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-16T09:19:55Z" title="Thursday, April 16th 2020, 11:19:55 am +02:00">Apr 16, 2020</time>_

---


Really deserves its own refactoring session/PR.
I got this from the storybook/ally-plugin hints tab. There are a lot more, I eventually didn't pursue, 'cause that would have involved touching/refactoring 2-3 other components.